examples/workflow/3_perform_fit.ipynb

```diff
@@ -0,0 +1,375 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pycompwa.ui as pwa"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "particle_list = pwa.read_particles('model.xml')\n",
+    "kinematics = pwa.create_helicity_kinematics('model.xml', particle_list)\n",
+    "kinematics.create_all_subsystems()\n",
+    "data_sample = pwa.read_root_data(input_file='generated_data.root')\n",
+    "phsp_sample = pwa.read_root_data(input_file='generated_phsp.root')\n",
+    "intensity_builder = pwa.IntensityBuilderXML(\n",
+    "    'model.xml', particle_list, kinematics, phsp_sample)\n",
+    "intensity = intensity_builder.create_intensity()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 3.1 Define estimator"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To perform a fit, you need to define an **estimator**. An estimator is a measure for the discrepancy between the intensity model and the data distribution to which you fit. To create an estimator we therefore generally need:"
+   ]
+  },
+  {
+   "cell_type": "raw",
+   "metadata": {
+    "raw_mimetype": "text/restructuredtext"
+   },
+   "source": [
+    "- an :class:`pycompwa.ui.Intensity` instance\n",
+    "- a :class:`pycompwa.ui.DataSet` to which we fit that intensity\n",
+    "- optionally: a phase space :class:`~.DataSet` if you\n",
+    "  want to normalize the :class:`~.Intensity`"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The data sample of momentum tuples can be converted to a `DataSet` as follows:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "data_set = kinematics.convert(data_sample)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now we can create an estimator. In this example, we use an **unbinned log likelihood** estimator. This estimator is used most commonly in high-energy physics."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "estimator, initial_parameters = \\\n",
+    "    pwa.create_unbinned_log_likelihood_function_tree_estimator(\n",
+    "        intensity, data_set)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Notice that you not only receive a estimator object, but also a list of fit parameters. You use this list of fit parameters to initialize the optimizer later on.\n",
+    "\n",
+    "The parameters contains the following info:\n",
+    "- initial value\n",
+    "- whether they are free or fixed\n",
+    "- optional upper and lower limits\n",
+    "- errors that give certain optimizers hints on the step size"
```

Comment:
Not rendered as a list

Reply:
https://github.com/ComPWA/pycompwa/pull/74/commits/c8a49675e989f3ec982584cbd916812bc948dccf